modules/unsupported/geojson-core/src/main/java/org/geotools/data/geojson/GeoJSONReader.java

```diff
@@ -240,6 +244,33 @@ public SimpleFeatureCollection getFeatures() throws IOException {
                     features.add(feature);
                 }
             }
+            // support paged feature collections, OGC API style
+            else if ("links".equalsIgnoreCase(name)) {
+                token = parser.nextToken();
+                if (!JsonToken.START_ARRAY.equals(token) || token == null) {
+                    break;
+                }
+                while (parser.nextToken() == JsonToken.START_OBJECT) {
+                    ObjectNode node = mapper.readTree(parser);
+                    JsonNode rel = node.get("rel");
+                    if (rel != null && "next".equals(rel.textValue())) {
+                        JsonNode href = node.get("href");
+                        if (href != null) {
+                            next = new URL(href.textValue());
+                            break;
+                        }
+                    }
+                }
+            }
+            // checking matched attributes from STAC/Features to optimize out size() if possible
+            else if ("context".equals(name) && parser.nextToken() == JsonToken.START_OBJECT) {
```

Comment:
everywhere else we've used `equalsignorecase` rather than just `equals` - which may be paranoia or justified, but we should probably do that here, or just do a `toLower()` on name when creating it and using `equals` through out.

Reply:
JSON is case sensitive but... why not. I'll make the change.

Reply:
Done, and merging.